$NetBSD: patch-ac,v 1.3 2010/02/24 12:37:04 martti Exp $

Use the package maintainer as the source of this problem.

--- bin/fvwm-bug.in.orig	2010-02-24 08:37:14.000000000 +0200
+++ bin/fvwm-bug.in	2010-02-24 08:38:08.000000000 +0200
@@ -130,7 +130,7 @@
 
 
 cat > $TEMP <<EOF
-From: ${USER}
+From: @MAINTAINER@
 To: ${BUGADDR}
 Subject: [50 character or so descriptive subject here (for reference)]
 
